--- chrooted_cleaner_script.sh
+++ chrooted_cleaner_script.sh.patched
@@ -204,7 +204,6 @@
         clonezilla
         gpart
         gparted
-        grsync
         hdparm

         # ENDEAVOUROS REPO
@@ -215,8 +214,6 @@
         ## Calamares EndeavourOS
         $(pacman -Qq | grep calamares)        # finds calamares related packages
         ckbcomp
-        kvantum
-        qt5ct

         # arm qemu dependency
         qemu-arm-aarch64-static-bin
